js/apps/system/_api/foxx/APP/index.js

```diff
@@ -114,7 +115,7 @@ if (FoxxManager.isFoxxmaster()) {
     const service = FoxxManager.lookupService(mount);
     res.json(serviceToJson(service));
   })
-  .body(schemas.service, ['multipart/form-data', 'application/zip', 'application/javascript', 'application/json'], `Service to be installed.`)
+  .body(schemas.service, ['application/javascript', 'application/zip', 'multipart/form-data', 'application/json'], `Service to be installed.`)
```

Comment:
Having multipart first means it is the default request body types. It's safer to have one of the types that are treated as an opaque buffer instead because this means we'll do the right thing if JS is uploaded as `text/plain` or zip as `application/octet-stream` or some other silly MIME mistakes are made.